src/useLifecycleLogger/__docs__/story.mdx

```diff
@@ -0,0 +1,33 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+import { Example } from './example.stories';
+import { ImportPath } from '../../storybookUtil/ImportPath';
+
+<Meta title="Lifecycle/useLifecycleLogger" component={Example} />
+
+# useLifecycleLogger
+
+Console log
```

Comment:
Why the blockquote here and "Console log"?

Just copy-paste here description from hook's JSDoc

Reply:
Is this what you referring to? JSDocs?
https://github.com/streamich/react-use/blob/master/docs/useLogger.md

Reply:
No, the think you write within coments before function definition
```
/**
*/
```

Reply:
Got you.